guake/guake_app.py

```diff
@@ -101,14 +101,25 @@ class Guake(SimpleGladeApp):
     def __init__(self):
         def load_schema():
             log.info("Loading Gnome schema from: %s", SCHEMA_DIR)
+            with open(Path(SCHEMA_DIR, "version"), encoding="ascii") as version:
```

Comment:
Not sure if this is a good idea, when installed on the system I remember well all gschema can be placed into a single directory.

maybe the name of the schéma file + version. Or burn the guake version into the schéma file in a new key

Reply:
Didn't consider using a param in the schema file. Changed approach to use that. No additional version tracking files now.

Reply:
perfect !